docs/tasks/administer_cluster_quotas.md

```diff
@@ -174,40 +174,40 @@ Write the manifest for the ClusterQueue that references the flavors. It should
 look similar to the following:
 
 ```yaml
-# cluster-total.yaml
+# cluster-queue.yaml
 apiVersion: kueue.x-k8s.io/v1alpha2
 kind: ClusterQueue
 metadata:
-  name: cluster-total
+  name: cluster-queue
 spec:
   namespaceSelector: {}
   resources:
   - name: "cpu"
     flavors:
-    - name: x86
+    - name: x86-flavor
       quota:
         min: 9
-    - name: arm
+    - name: arm-flavor
       quota:
         min: 12
   - name: "memory"
     flavors:
-    - name: default
+    - name: default-flavor
       quota:
         min: 84Gi
 ```
 
 The flavor names in the fields `.spec.resources[*].flavors[*].resourceFlavor`
 should match the names of the ResourceFlavors created earlier.
 
-Note that `memory` is referencing the `default` flavor created in the [single flavor setup.](#single-clusterqueue-and-single-resourceflavor-setup)
-This means that you don't want to distinguish if the memory is given from `x86`
-or `arm` nodes.
+Note that `memory` is referencing the `default-flavor` flavor created in the [single flavor setup.](#single-clusterqueue-and-single-resourceflavor-setup)
```

Comment:
Actually, we probably want to change this.

I think I wrote it before we supported codependent resources https://github.com/kubernetes-sigs/kueue/blob/main/docs/concepts/cluster_queue.md#codependent-resources

typically cpu and memory are tied to the same flavors.

Reply:
@alculquicondor can you clarify this, please?

Reply:
So before codependent resources where supported, the example that I put avoided having flavors for memory.

Now that they are supported, it's better to just put the same flavors for memory as cpu, because in real-world clusters, CPU and memory are usually resources tied to machine types or spot/on-demand.